soweego/cli.py

```diff
@@ -23,7 +23,7 @@
 
 # set env variable to ignore tensorflow warnings
 # (only errors are printed)
-tf.logging.set_verbosity(tf.logging.ERROR)
+tf.get_logger().setLevel(logging.ERROR)
```

Comment:
This seems to be not working, see https://github.com/Wikidata/soweego/pull/384#issuecomment-600806421